### PR TITLE
Pakhai hw 4

### PR DESCRIPTION
**Pull request by ViktorPakhai**
Hallo! My home work #4!

### CODE REVIEW

File: pakhai/HW3/task_4.py [hunk 1]
```diff
@@ -5,4 +5,11 @@
...
-print("Perimeter of the circle: ", radius * (math.pi *2) )
\ No newline at end of file
+print("Perimeter of the circle: ", radius * (math.pi *2) )
+
+
+
+
+
+
+a_dict = {'1': 'one', '2': 'two', '3': 'three'}
```
Comment (katemalash) [inline pakhai/HW3/task_4.py:15]: Is this dict needed here along with 6 empty lines before it?

File: pakhai/hw4/task_1.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# The YEAR is known. Determine whether this year is intercalary and to what century this year belongs
+
+year = int(input("Please, enter a year:  "))
+
+if year < 1582:
+    print("The calculation of leap years began after 1582. Enter another year!  ")
+else:
+    if   (year % 400) == 0 and (year % 4 ) == 0:
```
Comment (katemalash) [inline pakhai/hw4/task_1.py:8]: Are you sure this is correct condition?

File: pakhai/hw4/task_2.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+if figure == 1:
+    side_A = int(input("What lengt side A of rectangle: "))
+    side_B = int(input("What lengt side B of rectangle: "))
+    print("The square of ​​a rectangle: ", side_A * side_B)
+elif figure == 2:
+    a = int(input("What lengt side a of triangle: "))
+    b = int(input("What lengt side b of triangle: "))
+    c = int(input("What lengt side c of triangle: "))
+    p = (a + b + c) / 2
+    s = float('%.6f' % math.sqrt(p*(p - a)*(p - b)*(p - c)))
```
Comment (katemalash) [inline pakhai/hw4/task_2.py:19]: Just advice: braces have higher priority, so `s = float('%.6f' % math.sqrt(p * (p-a) * (p-b) * (p-c)))`

File: pakhai/hw4/task_2.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    print ("The square of ​​a triangle: ", s)
+elif figure == 3:
+    radius = int(input("Enter the radius of the circle:  "))
+    print("Area of the circle: ", math.pi* (radius ** 2))
```
Comment (katemalash) [inline pakhai/hw4/task_2.py:23]: Please add space before "*" operator

File: pakhai/hw4/task_4.py [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+squareHall= int(input("Enter the square of hall: "))
+radiuStage= int(input("Enter the radius of stage: "))
+pass_K=int(input("Enter the pass between the stage and wall: "))
+
+sideHall = math.sqrt(squareHall)
+
+if ( sideHall-2*radiuStage >= pass_K):
+    print("OK")
+else:
+    print("Sorry! It is not possible!")
```
Comment (katemalash) [inline pakhai/hw4/task_4.py:15]: Pretty nice!

File: pakhai/hw4/task_5.py [hunk 6]
```diff
@@ -0,0 +1,10 @@
+# The amount of money is known. Exchange it with 200, 100, 10 banknotes and 1 UAH coin, if possible
+
+money=int(input("Please enter the summ of monney: "))
+
+coin200 = money//200 
+coin100 = (money - coin200*200)//100
```
Comment (katemalash) [inline pakhai/hw4/task_5.py:6]: `coin100 = (money - coin200*200) // 100`

File: pakhai/hw4/task_5.py [hunk 7]
```diff
@@ -0,0 +1,10 @@
...
+coin10 = (money - coin200*200 - coin100*100)//10
+coin1 = (money - coin200*200 - coin100*100 -coin10*10)//1
```
Comment (katemalash) [inline pakhai/hw4/task_5.py:8]: Is this "//1" necessary?

File: pakhai/hw4/task_5.py [hunk 8]
```diff
@@ -0,0 +1,10 @@
...
+
+print ("Count 200: ",coin200 ,"\nCount 100: ",coin100, "\nCount 10: ", coin10, "\nCount 1: ",coin1 )
```
Comment (katemalash) [inline pakhai/hw4/task_5.py:10]: In general, good decision.

File: pakhai/hw4/task_6.py [hunk 9]
```diff
@@ -0,0 +1,12 @@
+bilet = int(input("Enter the ticket number: "))
+
+if (bilet%2==0) and  ( bilet > 0) and ( bilet <= 36)  :
+    print ("Місце  в купе,  верхнє.")
+elif (bilet%2==0) and bilet > 36 and bilet<55:
```
Comment (katemalash) [inline pakhai/hw4/task_6.py:5]: just clarifying: this option `36 < bilet < 55` also works

File: pakhai/hw4/task_6.py [hunk 10]
```diff
@@ -0,0 +1,12 @@
...
+    print ("Місце в боковому секторі, вєрхнє.")
+elif (bilet%2==1) and bilet <= 36:
+    print ("Місце в купе секторі, нижнє.")
+elif (bilet%2==1) and bilet > 36 and bilet<55:
+    print ("Місце в боковому секторі, нижнє.")
+else:
+    print("Щось не так! Перевірте уважно місце в білеті!")
```
Comment (katemalash) [inline pakhai/hw4/task_6.py:12]: Good Decision

**Review by katemalash — APPROVED** [review]
In general home work is done good. You can merge this pull-request after fixing the minor issues.

File: pakhai/hw4/task_1.py (hunk 2 above)
Comment (ViktorPakhai) [reply pakhai/hw4/task_1.py:8]: Так! Код робочий!  підкажи де саме ти бачиш помилку? попробую пофіксити...

**Pull request merged**